app/views/organization_alliances/index.html.erb

```diff
@@ -0,0 +1,102 @@
+<h1><%= t('organization_alliances.title') %></h1>
+
+<div class="row">
+  <div class="col-12 col-sm-12 col-md-12 col-lg-12">
+    <ul class="nav nav-pills actions-menu">
+      <li class="nav-item">
+        <%= link_to organization_alliances_path(status: 'pending'), class: "nav-link #{'active' if @status == 'pending'}" do %>
+          <%= glyph :time %>
+          <%= t('organization_alliances.status.pending') %>
+        <% end %>
+      </li>
+      <li class="nav-item">
+        <%= link_to organization_alliances_path(status: 'accepted'), class: "nav-link #{'active' if @status == 'accepted'}" do %>
+          <%= glyph :ok %>
+          <%= t('organization_alliances.status.accepted') %>
+        <% end %>
+      </li>
+      <li class="nav-item">
+        <%= link_to organization_alliances_path(status: 'rejected'), class: "nav-link #{'active' if @status == 'rejected'}" do %>
+          <%= glyph :remove %>
+          <%= t('organization_alliances.status.rejected') %>
+        <% end %>
+      </li>
+      <li class="nav-item ms-auto">
+        <%= link_to organizations_path, class: "text-primary" do %>
+          <%= glyph :search %>
+          <%= t('organization_alliances.search_organizations') %>
+        <% end %>
+      </li>
+    </ul>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-md-12">
+    <div class="card">
+      <div class="card-body table-responsive">
+        <table class="table table-hover table-sm">
+          <thead>
+            <tr>
+              <th><%= t('organization_alliances.organization') %></th>
+              <th><%= t('organization_alliances.city') %></th>
+              <th><%= t('organization_alliances.members') %></th>
+              <th><%= t('organization_alliances.type') %></th>
+              <% if @status != 'rejected' %>
+                <th><%= t('organization_alliances.actions') %></th>
+              <% end %>
+            </tr>
+          </thead>
+          <tbody>
+            <% @alliances.each do |alliance| %>
+              <% is_sender = (alliance.source_organization_id == current_organization.id) %>
+              <% other_org = is_sender ? alliance.target_organization : alliance.source_organization %>
```

Comment:
move this code to a helper, the views should have the less ruby code as possible